GooglePlayInstant/Editor/PlayInstantLoadingScreenGenerator.cs

```diff
@@ -21,20 +22,28 @@
 
 namespace GooglePlayInstant.Editor
 {
-    //TODO: Resolve conversation about upload vs setting path
     /// <summary>
     /// Class that generates Unity loading scenes for instant apps.
     /// </summary>
     public class PlayInstantLoadingScreenGenerator
     {
         public const string LoadingSceneName = "play-instant-loading-screen-scene";
 
-        public static string loadingScreenImagePath;
-        
-        //TODO: add documentation
+        private const string LoadingScreenJsonFileName = "LoadingScreenConfig.json";
+
+        private static readonly string LoadingScreenScenePath =
+            Path.Combine("Assets", "PlayInstantLoadingScreenAssets");
+
+        private static readonly string LoadingScreenJsonPath = Path.Combine(LoadingScreenScenePath, "Resources");
+
+        public static string LoadingScreenImagePath;
+
+        /// <summary>
+        /// Sets the loadingScreenImagePath target for future generated loading screens.
+        /// </summary>
         public static void SetLoadingScreenImagePath()
         {
-            loadingScreenImagePath =
+            LoadingScreenImagePath =
                 EditorUtility.OpenFilePanel("Select Image", "", "png,jpg,jpeg,tif,tiff,gif,bmp");
```

Comment:
Other than right here, this class doesn't do anything GUI related. Maybe OpenFilePanel should be called from where the button is that launches it.